aiidalab_widgets_base/codes.py

```diff
@@ -2,7 +2,30 @@
 
 from __future__ import absolute_import
 import ipywidgets as ipw
+
+from subprocess import check_output
 from IPython.display import clear_output
+from aiida import load_dbenv, is_dbenv_loaded
+from aiida.backends import settings
+if not is_dbenv_loaded():
+    load_dbenv(profile=settings.AIIDADB_PROFILE)
+from aiida.orm import Code, Computer
```

Comment:
`Computer` is only used for `_get_codes`. The import could be moved there.

Reply:
done